### PR TITLE
Shultsedu 287 map location tracking background

### PR DESCRIPTION
**Pull request by nadiiazhurba**

### CODE REVIEW

File: SideMenuFlyoutShellTestProject.Android/Services/GeolocationService/LocationService.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+[assembly: Dependency(typeof(LocationService))]
+namespace SideMenuFlyoutShellTestProject.Droid.Services.GeolocationService
+{
+    public class LocationService : SharedLocationService
+    {
+        public override void StartTrackingLocation(int frequencySeconds = 10)
+        {
+            var action = new Action(delegate { StartTrackingTimer(frequencySeconds); });
+            MessagingCenter.Send(this, "StartTrackingLocation", action);
```
Comment (YuliaLoyko) [inline SideMenuFlyoutShellTestProject.Android/Services/GeolocationService/LocationService.cs:17]: let's not hardcode strings)

File: SideMenuFlyoutShellTestProject.iOS/AppDelegate.cs [hunk 2]
```diff
@@ -25,5 +26,21 @@ public override bool FinishedLaunching(UIApplication app, NSDictionary options)
 
             return base.FinishedLaunching(app, options);
         }
+
+        public override void OnActivated(UIApplication uiApplication)
```
Comment (YuliaLoyko) [inline SideMenuFlyoutShellTestProject.iOS/AppDelegate.cs:30]: could you please check this article (and may some others) about background modes for iOS?
let's implement system background mode)
 https://developer.apple.com/documentation/corelocation/getting_the_user_s_location/handling_location_events_in_the_background

**Review by YuliaLoyko — CHANGES_REQUESTED** [review]

File: SideMenuFlyoutShellTestProject.Android/Services/GeolocationService/LocationService.cs (hunk 1 above)
Comment (nadiiazhurba) [reply SideMenuFlyoutShellTestProject.Android/Services/GeolocationService/LocationService.cs]: Done it

File: SideMenuFlyoutShellTestProject.iOS/AppDelegate.cs (hunk 2 above)
Comment (YuliaLoyko) [reply SideMenuFlyoutShellTestProject.iOS/AppDelegate.cs:30]: so did you add location background mode? i don't see changes in info.plist

**Review by YuliaLoyko — APPROVED** [review]

**Pull request merged**